editor/src/components/editor/actions/actions.ts

```diff
@@ -3584,6 +3582,12 @@ export const UPDATE_FNS = {
     userState: UserState,
   ): EditorModel => {
     const file = getContentsTreeFileFromString(editor.projectContents, action.filename)
+
+    // Don't delete package.json, otherwise it will bring about the end of days.
```

Comment:
😹

Reply:
Ha this comment was already in the code, we had just moved the associated line up a few lines